### PR TITLE
Aws instance provision

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
This PR is targeting the task of deployment an OCP-3.x cluster on AWS.
The implementation is utilizing [openshift-ansible](https://github.com/openshift/openshift-ansible/tree/release-3.11) and is inspired by [these](https://gitlab.sat.engineering.redhat.com/rhci/vm_env/blob/master/openshift/aos-3.11/) roles.

### CODE REVIEW

File: roles/ocp3_aws_provision/defaults [hunk 1]
```diff
@@ -0,0 +1 @@
+../ocp3_aws_deploy/defaults/
```
[the remark this answers is not in the document]
Comment (Danil-Grigorev) [reply roles/ocp3_aws_provision/defaults]: This one is a symlink, which is pointing to the `roles/ocp3_aws_deploy/defaults/` directory. The reason is to have all the variables for both roles in one file, and don't bother with using `include_vars` in the second one.

File: get-openshift-ansible.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- hosts: localhost
```
[the remark this answers is not in the document]
Comment (Danil-Grigorev) [reply get-openshift-ansible.yml]: Yep, sounds better, will fix it.

File: roles/ocp3_aws_provision/defaults (hunk 1 above)
[the remark this answers is not in the document]
Comment (fbladilo) [reply roles/ocp3_aws_provision/defaults]: @Danil-Grigorev Symlinks are not good practice for ansible, for this we have some options : 

1.  We use a variables file that contains common variables used by both roles, we could even reuse config/defaults.yml in this case or if you prefer we can create an ocp3_defaults.yml just for this. We can then use vars_files at the play level to ensure they get sourced.
2. We use a include_vars on the second role calling {{playbook_dir

[the remark this answers is not in the document]
Comment (Danil-Grigorev) [reply roles/ocp3_aws_provision/defaults]: Fixed.

**Pull request merged**